### PR TITLE
Fix morgue trays flying away

### LINKED ISSUE #2540: Opening a morgue in Zero-G makes the slab float away until it hits something solid.
How to reproduce: Open one of the morgues in while the gravity generator is busted, I guess.

The morgues can still be closed normally, making the slab vanish from the other side of the room.
I didn't have time to test wether in that case an item that's on the morgue slab tile or in front of the morgue itself is gathered up, so for all intents and purposes, it might just be a graphic bug.

### PR DESCRIPTION
**Pull request by tigercat2000**
Closes #2540

### CODE REVIEW

**Comment by Jovaniph** [comment]
That'll fix it for sure.

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -183,7 +183,7 @@
  */
 /obj/structure/m_tray
 	name = "morgue tray"
-	desc = "Apply corpse before closing."
+	desc = "Apply corpse before closing. May float away in no-gravity."
```
Comment (Crazylemon64) [inline code/game/objects/structures/morgue.dm]: 👌

**Comment by taukausanake** [comment]
I was just wondering, can you pull trays? It would be better if you can pull them back into place or even then move them around like a ghetto roller bed. Just a passing thought

**Comment by Anticept** [comment]
If you click the morgue housing, it instantly retracts them, no matter how far away they have floated.

**Comment by SamHPurp** [comment]
> If you click the morgue housing, it instantly retracts them, no matter how far away they have floated.

Its a joke PR.

**Comment by Anticept** [comment]
I know it's a joke PR, I was replying to tau's question.

**Comment by taukausanake** [comment]
> tau's

There are too many definitions for that word and I feel bad about being any of them. Especially a shitty pi replacement

**Comment by Anticept** [comment]
HAHAHAHAHA i love that this is merged.

**Comment by SamHPurp** [comment]
....

**Comment by SamHPurp** [comment]
Yes because it floating umpteen tiles away, but insta-teleporting back into the container is Working As Intended :tm:

**Comment by tigercat2000** [comment]
@SamHPurp I guess we could add a lightning bolt effect if the tray is more than a tile away :thinking:

**Comment by SamHPurp** [comment]
Bluespace magic

**Pull request merged**